components/footer.js

```diff
@@ -0,0 +1,45 @@
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faFacebookF, faTwitter, fab } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import '@fortawesome/fontawesome-svg-core/styles.css';
+import css from '../styles/footer.module.scss';
+
+library.add(fab, faFacebookF, faTwitter);
+
+export function Footer({ children }) {
+    const CURRENT_YEAR = new Date().getFullYear();
+    return (
+        <>
+            <footer className={css.footer}>
+                <div className={css.socialmedia}>
+                    <div className={css.facebook}>
+                        <a
+                            href="https://www.facebook.com/ab50"
```

Comment:
If we do include any footers, it will be the common www.aweandreverence.com footers, and the social links will be to Awe and Reverence. We will not be creating separate AB506 social profiles.

Reply:
done